caddyhttp/maxrequestbody/maxrequestbody_test.go

```diff
@@ -0,0 +1,83 @@
+package maxrequestbody
+
+import (
+	"testing"
+
+	. "github.com/mholt/caddy/caddyhttp/httpserver"
```

Comment:
Not easy. 100 * KB is more readable than say 100 * validUnits["KB"] or 100 * validUnits[1], depending on whether I'm making it a map or a list.

Also I traverse the units in order in the parser code so if I were to rewrite validUnits to a map I would need to make an additional variable to store the order of the keys.

I would keep the constants.